### DISCUSSION TITLE
white spaces after bootstrap links

### DISCUSSION THREAD

**Discussion by fabioprandi** [twbs/bootstrap#39956] [Q&A]
Hi, i'm new with bootstrap and i'm having an issue. 
When i add the css and js bootstrap links inside my html, it also add white spaces and i don't know how to remove them. I tryed to put in my .body a background color with !important but it doesn't work

**Comment by julien-deramond** [comment]
Hi @fabioprandi 
Could you maybe provide some code or create a reproducible environment with CodePen or StackBlitz so that someone from the community could help?

**Reply by fabioprandi** [reply]
sure thanks, i'm adding these links from bootstrap:

```html
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">

<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
```

and in my css i have:

```css
body {
background-color: #6cc4f0 !important;
}
```

---

_Edited by a maintainer for a correct syntax highligthing_

**Comment by bryanj408** [comment]
Hi @fabioprandi 
You can go to https://codepen.io/pen/ and put in some more information to go off of. I've attached a screenshot of where you would put in your CDN links in the settings section. Codepen is a really powerful tool to see your results in real time and troubleshoot. Great way to share issues you're having like @julien-deramond pointed out. 
<img width="542" alt="codepen" src="https://github.com/twbs/bootstrap/assets/38339891/f930bf0c-bf0c-4087-90df-f57f518c9afb">
